edx/edx-enterprise
# [ENT-157] Reconfigure isort linter

**Description:** This PR sets the isort linter to use the [Vertical Hanging Indent mode](https://github.com/timothycrosley/isort#multi-line-output-modes) with trailing comma support to make imports easier to insert and review.

**JIRA:** [ENT-157](https://openedx.atlassian.net/browse/ENT-157)

**Dependencies:** None

**Testing instructions:**

1. Check out this branch.
2. Run `make quality` and verify that the isort command (commands[12]) succeeds.
3. Optionally you can run `make test-all` to verify the whole suite passes.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Reviewers:**
